eng/Dependencies.props

```diff
@@ -79,6 +79,12 @@ and are generated based on the last package release.
     <LatestPackageReference Include="System.Threading.Channels" />
     <LatestPackageReference Include="System.ValueTuple" />
 
+    <!-- Runtime packages referenced only for exclusions -->
+    <LatestPackageReference Include="Microsoft.Win32.SystemEvents" />
+    <LatestPackageReference Include="System.Drawing.Common" />
+    <LatestPackageReference Include="System.Security.Permissions" />
+    <LatestPackageReference Include="System.Windows.Extensions" />
```

Comment:
Need the three of these that aren't already in eng/Version.Details.xml to either use `5.0.0` versions (and move out of the automated section of eng/Versions.props) or be included in eng/Version.Details.xml.

Reply:
These all still build live, so they can be included in the main section.  These should all be temporary until https://github.com/dotnet/runtime/issues/54341 is addressed.  IMHO it's worth carrying this package for one more release to make the type move.  I'd like to hear @ViktorHofer's thoughts.